arctic/store/_pickle_store.py

```diff
@@ -54,12 +56,16 @@ def write(self, arctic_lib, version, symbol, item, previous_version):
         collection = arctic_lib.get_top_level_collection()
         # Try to pickle it. This is best effort
         version['blob'] = _MAGIC_CHUNKED
-        pickled = lz4.compressHC(cPickle.dumps(item, protocol=cPickle.HIGHEST_PROTOCOL))
+        try:
+            pickled = lz4.compressHC(cPickle.dumps(item, protocol=cPickle.HIGHEST_PROTOCOL))
+        except OverflowError:
```

Comment:
I was thinking: try to write data with Lz4x64, succeed, but end up with data that cannot be decompressed with Lz4x32.

(Feel free to overrule me if you don't think this is a worthwhile worry.)